.github/workflows/trigger-quantic-e2e-setup.yml

```diff
@@ -0,0 +1,40 @@
+name: Trigger Quantic E2E Setup
+
+on:
+  workflow_dispatch:
+
+  workflow_call:
+    secrets:
+      SFDX_AUTH_CLIENT_ID:
+        required: true
+      SFDX_AUTH_JWT_KEY:
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  e2e-quantic-setup:
+    name: 'Setup e2e tests on Quantic'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+        with:
+          egress-policy: audit
+
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+
+      - name: Project setup
+        uses: ./.github/actions/setup
+
+      - name: Setup Salesforce CLI
+        uses: ./.github/actions/setup-sfdx
+
+      - name: Setup Quantic E2E
+        uses: ./.github/actions/e2e-quantic-setup
+        with:
+          clientid: ${{ secrets.SFDX_AUTH_CLIENT_ID }}
+          jwtkey: ${{ secrets.SFDX_AUTH_JWT_KEY }}
```

Comment:
Can we try to make it a bit clearer to output the community value? Also @mmitiche playwright PR will change those, so depending who merge's first we should change this:

Example: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-output-parameter

```
- name: Set base community url
  id: community-url
  run: echo "COMMUNITY_URL=$BASE_URL" >> "$GITHUB_OUTPUT"
```

You can even make it look nice with markdown:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#example-of-adding-a-job-summary

```
- name: Output base community url
  run: |
    echo "Access Quantic Community here : $BASE_URL" >> $GITHUB_STEP_SUMMARY
```

Reply:
for the secrets, they seem unchanged in his [PR](https://github.com/coveo/ui-kit/pull/4711) no?

as for the output, I am trying a bunch of options, but it does not seem like the env set in the `deploy-community.ts` propagates all the way to the workflow level